1-bootstrap/main.tf

```diff
@@ -33,6 +33,11 @@ locals {
       bucket_infix = "fs"
       roles        = []
     }
+    "cymbalbank" = {
+      repo_name    = "eab-cymbalbank",
+      bucket_infix = "cb"
+      roles        = []
+    }
```

Comment:
Don't create the application source repos at this stage; they are created by the app factory

Reply:
ok, done.